website/content/docs/k8s/installation/vault/enterprise-license.mdx

```diff
@@ -80,6 +80,7 @@ use the enterprise license key in Vault:
 
 ```yaml
 global:
+  image: hashicorp/consul-enterprise:1.11.0-ent
```

Comment:
```suggestion
  image: hashicorp/consul-enterprise:1.11.4-ent
```

Reply:
We should use the latest image if possible.

Reply:
yeah agreed, I shoulda checked the latest.